overrides/config/ftbquests/quests/chapters/all_things_food.snbt

```diff
@@ -1350,6 +1383,25 @@
 					}
 					type: "item"
 				}
+				{
+<<<<<<< Updated upstream
+					id: "7088EECFE736BC24"
+=======
+					id: "2F283A5E4037145F"
+>>>>>>> Stashed changes
+					item: {
+						Count: 1b
+						ForgeCaps: {
+							"dungeons_libraries:built_in_enchantments": { }
+						}
+<<<<<<< Updated upstream
+						id: "enlightened_end:elevibloom_seeds"
+=======
+						id: "atmospheric:aloe_kernels"
+>>>>>>> Stashed changes
+					}
+					type: "item"
+				}
```

Comment:
these "<<<<<<< Updated upstream" lines are remenants of git merges, try removing it but double check that elevibloom seeds and aloe kernals still show up properly